tests/lit-tests/2280.ispc

```diff
@@ -0,0 +1,30 @@
+// RUN:     %{ispc} --target=host --nowrap --nostdlib -g -O0 -o %t.o %s -E
```

Comment:
A few comments here
`-g -O0` is not essential here, so no reason to use it.
`--nowrap` is needed only when you are matching the output for a specific error message, so it's not wrapped depending on the terminal settings.
`--nostdlib` is good to use, as it basically reduces compile time.

Reply:
Based on this, wouldn't is be safest to default to `--nowrap` so that terminal settings are never assumed (and therefore can be safely ignored)?

Reply:
Wrapping is good for human beings. They are bad for parsers.